crux-mir/translate_libs.sh

```diff
@@ -28,6 +28,46 @@ translate_2015() {
         "$@"
 }
 
+usage() {
+    echo "Usage: ${0} [FLAGS]"
+    echo ""
+    echo "Flags:"
+    echo "-a --model-internal-atomics      Include stdlib internal atomics in concurrency model (default)"
+    echo "   --no-model-internal-atomics   Do not include stdlib internal atomics in concurrency model"
+}
+
+SHOULD_MODEL_ATOMICS=1
+
+for arg in "$@"
+do
+    case $arg in
+        -a|--model-internal-atomics)
+            SHOULD_MODEL_ATOMICS=1
+            shift
+            ;;
+        --no-model-internal-atomics)
+            SHOULD_MODEL_ATOMICS=0
+            shift
+            ;;
+        -h|--help)
+            usage
+            exit 0
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+
+if [[ $SHOULD_MODEL_ATOMICS -eq "1" ]]; then
```

Comment:
`[[` is a bash builtin - either switch to `[` or change the shebang line from `/bin/sh` to `/bin/bash`